website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -0,0 +1,102 @@
+---
+layout: docs
+page_title: Consul K8s CLI Reference
+description: >-
+  Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
+---
+
+# Consul K8s CLI Reference
+
+Consul K8s CLI is a tool for quickly installing and interacting with Consul on Kubernetes.
+This topic describes the commands, subcommands, and available options for using Consul K8s CLI.
+
+~> Consul K8s CLI is is currently availabe as an alpha release and is not recommended for production environments.
+
+## Download and Build the Binary
+
+1. Clone the [consul-k8s-cli](https://github.com/hashicorp/consul-k8s-cli) project.
```

Comment:
We're close to finalizing the install PR into the consul-k8s repo itself, so as soon as that PR is merged, we'll want people to build it from there: (https://github.com/hashicorp/consul-k8s/tree/cli-install/cli). We should have install merged by alpha. We could totally merge this docs PR as is, and create a new docs PR to update this as soon as it's merged.

We'd just need to update this step and the step below to reference the correct paths (i.e to clone the consul-k8s repo instead, navigate to consul-k8s/cli, run go build from there.)

We may want to consider just having them do 
```
go build .
```
and adding an additional optional step to say:
(Optional) Consider moving the binary into your $PATH for ease of use:
```
mv consul-k8s /usr/local/bin/consul-k8s
```